src/app/components/work-item-create/work-item-create-selector/work-item-create-selector.component.less

```diff
@@ -1,12 +1,11 @@
 @import (reference) '../../../../assets/stylesheets/_base';
-
 .f8-wi-modal {
   position: absolute;
   top: 0;
   left: 0;
   right: 0;
   bottom: 0;
-  z-index: 3;
+  z-index: 9999;
   padding: (@grid-gutter-width/2);
```

Comment:
@mindreeper2420 can you confirm the z-index value?

Reply:
All modals should be pre-determined by the frameworks. If something has to be set at this high of an amount, then there are other factors that need to be addressed.

The story that this fixes does not have a lot of information as to what the problem is.